### PR TITLE
Select OpenMusic by default when opening a new scenario

### PR DESCRIPTION
**Pull request by karst**
As requested on Reddit. Idk how people feel about this, feel free to discuss.

Depends on https://github.com/OpenRCT2/OpenRCT2/issues/19827

### CODE REVIEW

**Comment by karst** [comment]
idk why macos is failing, but i am pretty certain it doesn't have anything to do with this pr.

**Comment by IntelOrca** [comment]
macOS tests are failing as it doesn't seem to have the OpenRCT2 music objects when running the tests.

**Comment by karst** [comment]
Guess a fix for that would have to be merged first eh

**Comment by karst** [comment]
Converting to draft until #19827 has been fixed

**Comment by karst** [comment]
Keep open please

**Comment by karst** [comment]
Keep open please

File: src/openrct2/rct12/RCT12.cpp [hunk 1]
```diff
@@ -750,6 +750,14 @@ static constexpr std::string_view _musicStyles[] = {
     "rct2.music.pirate",
     "rct2.music.rock3",
     "rct2.music.candy",
+    "openrct2.music.galaxy",
```
Comment (duncanspumpkin) [inline src/openrct2/rct12/RCT12.cpp:753]: This will only add it for sc6/sc4 scenarios what about .park scenarios?

Comment (karst) [reply src/openrct2/rct12/RCT12.cpp:753]: I'll look into it! Might be smart to do it there, too, tho I think as long as it's part of the basic selection when creating a scenario it should be fine, too.

**Comment by karst** [comment]
I would still like to see this implemented

**Comment by karst** [comment]
Let me know when I need to rebase this.

File: src/openrct2/rct12/RCT12.cpp (hunk 1 above)
Comment (karst) [reply src/openrct2/rct12/RCT12.cpp:753]: I have implemented it to be part of any sc6/sc4 that gets opened, and it is now part of the default selection when creating a new .park scenario. It does not alter existing .park scenarios. For official scenarios the list should be updated in the scenario itself.

**Comment by karst** [comment]
Would be nice for OpenMusic to have this in before next release. Should be ready.

 Once this is in I'll update guidelines for adding new songs/creating new OpenMusic releases.

**Review by ZehMatt — APPROVED** [review]

**Review by Broxzier — APPROVED** [review]
Tested the build, and everything seems to be in order

**Pull request merged**